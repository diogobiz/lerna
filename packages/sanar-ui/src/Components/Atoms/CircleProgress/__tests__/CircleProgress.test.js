import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESCircleProgress from '../'

it('renders correctly', () => {
    const component = <ESCircleProgress percent={50} />

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
