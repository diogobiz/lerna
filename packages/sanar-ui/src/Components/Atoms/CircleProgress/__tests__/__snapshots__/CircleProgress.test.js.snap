// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ant-progress ant-progress-circle ant-progress-status-normal ant-progress-show-info ant-progress-default es-circle-progress es-circle-progress--normal es-circle-progress--undefined"
>
  <div
    className="ant-progress-inner"
    style={
      Object {
        "fontSize": 24,
        "height": 120,
        "width": 120,
      }
    }
  >
    <svg
      className="ant-progress-circle "
      style={Object {}}
      viewBox="0 0 100 100"
    >
      <path
        className="ant-progress-circle-trail"
        d="M 50,50 m 0,-47
   a 47,47 0 1 1 0,94
   a 47,47 0 1 1 0,-94"
        fillOpacity="0"
        stroke="#f3f3f3"
        strokeLinecap="square"
        strokeWidth={6}
        style={
          Object {
            "stroke": "#f3f3f3",
            "strokeDasharray": "295.3097094374406px 295.3097094374406px",
            "strokeDashoffset": "-0px",
            "transition": "stroke-dashoffset .3s ease 0s, stroke-dasharray .3s ease 0s, stroke .3s, stroke-width .06s ease .3s",
          }
        }
      />
      <path
        className="ant-progress-circle-path"
        d="M 50,50 m 0,-47
   a 47,47 0 1 1 0,94
   a 47,47 0 1 1 0,-94"
        fillOpacity="0"
        stroke=""
        strokeLinecap="square"
        strokeWidth={6}
        style={
          Object {
            "stroke": "#108ee9",
            "strokeDasharray": "147.6548547187203px 295.3097094374406px",
            "strokeDashoffset": "-0px",
            "transition": "stroke-dashoffset .3s ease 0s, stroke-dasharray .3s ease 0s, stroke .3s, stroke-width .06s ease .3s",
          }
        }
      />
    </svg>
    <span
      className="ant-progress-text"
      title="50%"
    >
      50%
    </span>
  </div>
</div>
`;
